README.md

```diff
@@ -1 +1,19 @@
-# Fill me in with information
+Official MongoDB Node.js Driver Quick Start
+===========================================
+
+You can use this starter project to follow along the
+[MongoDB Node.js Driver Quick Start](https://docs.mongodb.com/drivers/node/current/quick-start/).
+
+This project contains a `package.json` file which contains information
```

Comment:
Comment: I believe "that" should be used here as "contains information..." seems like a [defining clause](https://www.grammarly.com/blog/which-vs-that/?gclid=Cj0KCQiAosmPBhCPARIsAHOen-OXV5y40s2SiNFjQvdwlxw1SBepDoBIpzVvZ-shsCGAcFWtA-uMULoaAmkVEALw_wcB&gclsrc=aw.ds).

```suggestion
This project contains a `package.json` file that contains information
```

Reply:
Good catch. In the style guide, we refer to them as restrictive & non-restrictive clauses.